### PR TITLE
Add `-portable=false` option in Tizen CI build

### PR DESCRIPTION
**Pull request by jyoungyun**
Add `-portable=false` option in Tizen CI build

### CODE REVIEW

**Comment by jyoungyun** [comment]
Related issue : https://github.com/dotnet/coreclr/issues/11248

/cc @gkhanna79 @weshaggard

File: cross/arm32_ci_script.sh [hunk 1]
```diff
@@ -235,9 +235,10 @@ function cross_build_corefx_with_docker {
     # Cross building corefx with rootfs in Docker
     if [ "$__buildArch" == "armel" ]; then
         __extraCmd="/p:OverridePackageSource=https://tizen.myget.org/F/dotnet-core/api/v3/index.json"
+        __portableLinux="-portable=false"
```
Comment (gkhanna79) [inline cross/arm32_ci_script.sh]: @weshaggard Is there any such flag (-portable) that CoreFX supports?

Comment (weshaggard) [inline cross/arm32_ci_script.sh]: Yes build.sh does support this flag and it it is set it essentially stops passing "-portable" to the native builds and set PortableBuilds=false in the manage builds.

**Comment by weshaggard** [comment]
If you want to make corefx use PortableBuild as the option you can simply update https://github.com/dotnet/corefx/blob/master/config.json#L249, https://github.com/dotnet/corefx/blob/master/config.json#L389 and https://github.com/dotnet/corefx/blob/master/config.json#L542 to rename "portable" to "PortableBuild" and that should make that the corefx option as well.

**Comment by weshaggard** [comment]
@jyoungyun while I believe this works for the managed builds I don't believe it is doing what you expect for the native builds. Can you please verify that the native assets are correctly non-portable with this change?

**Pull request merged**